packages/fxa-auth-server/lib/senders/emails/templates/postAddRecoveryPhone/includes.json

```diff
@@ -6,5 +6,9 @@
   "action": {
     "id": "postAddRecoveryPhone-action",
     "message": "Manage account"
+  },
+  "preview": {
+    "id": "postAddRecoveryPhone-preview",
+    "message": "Account protected by two-step authentication"
```

Comment:
Thank you for cleaning up the mess I apparently left :sweat_smile:

Reply:
It was just small things! + the option to add previews is new so this is an enhancement ✨